### DISCUSSION TITLE
Qubino Virtual Switch (Input 2 Switch) missing as Device when enabling Configuration Setting 100 [Solved]

### DISCUSSION THREAD

**Discussion by techsurrusr** [zwave-js/zwave-js-ui#3526] [Q&A] [answered]
Does zwavejsui support the usage of Qubino Virtual Switch (Input 2 Switch) which can be used to turn on a specific group/device? Configuration settings '100' allows to set the 'Enable / Disable Endpoints I2' to 'Binary Sensor' but not even sure if this is correct or whether it's supported by z2jsui using virtual switches?

**Comment by robertsLando** [comment, marked as answer]
Question is not if ZUI supports that, question is if the device does respect the protocol when doing that, maybe you have to exclude and include again the device or re-interview it when you change that settings

cc @AlCalzone

**Comment by AlCalzone** [comment]
Yeah that works. You'll have to exclude and re-include after that change, like the device manual states.

**Reply by techsurrusr** [reply]
Thanks, excluding and including did the trick to have the additionally binary switch to show up. @AlCalzone would you know if any of the other settings for parameter 100 could be used to mimic mono-type switch as I'm using this for I1 and would be great to have both switches working in the same way.

**Reply by techsurrusr** [reply]
Found the answer and it's not possible to use any of the other parameter settings to mimic mono-switch, so the I2 and I3 can only behave as binary switch.
